autorest/codegen/templates/model.py.jinja2

```diff
@@ -37,7 +37,7 @@ class {{ model.name }}({{ basename }}):
 {% else %}
 class {{ model.name }}(Model):
 {% endif %}
-    """{{ model.description }}
+    """{{ model.description | wordwrap(width=80, break_long_words=False) | replace('\n', '\n\t') }}
```

Comment:
We generate a space indented file, not tab indented file. Python does not allow anymore mixed tab and space.
I think that probably should be 5 spaces.
Also you should be able to use the wrapstring parameter of wordwrap, and not a replace (maybe, to be tested)
https://jinja.palletsprojects.com/en/2.10.x/templates/#wordwrap